packages/server/test/items/itemSmashable.test.ts

```diff
@@ -0,0 +1,137 @@
+//         //     WorldScene: class MockWorldScene { },
+//         // }));
+
+
+//         test('Test zero damage hit', () => {
+
+//             const { testSmashable: smashable } = jest.requireActual('../../src/items/smashable');
+
+//             testSmashable = jest.fn().mockImplementation((attackDmg: number) => {
+//                 return {
+
+//                     smashable
+
+//                 };
+//             });
+
+//             const fenceMaxHealth = 100;
+
+//             let testFence = new SpriteItem(fenceMaxHealth, fenceMaxHealth);
+//             testFence.
+
+//             // expect(fullHealthFence.isBelowMaxHealth()).toBe(false);
+//             // expect(fullHealthFence.calculateHealthPercentage()).toBe(1);
+
+//             // const halfHealthFence = new SpriteItem(fenceMaxHealth, fenceMaxHealth / 2);
+//             // expect(halfHealthFence.isBelowMaxHealth()).toBe(true);
+//             // expect(halfHealthFence.calculateHealthPercentage()).toBe(0.5);
+
+//         });
+
+//         test('Test health bar in sprite_item constructor', () => {
+//             jest.requireActual('../../../src/sprite/sprite_item');
+
+//             SpriteItem = jest.fn().mockImplementation((
+//                 maxHealth: number, health: number, scene: Phaser.Scene, itemType: { layoutType: string }
+//             ) => {
+//                 const sprite = {
+//                     attributes: { health },
+//                     itemType,
+//                     healthBar: undefined as Phaser.GameObjects.Graphics | undefined,
+//                     maxHealth: undefined as number | undefined,
+//                     intialize() {
+//                         // Copied From SpriteItem Constructor
+//                         if (itemType.layoutType === 'fence' || itemType.layoutType === 'wall') {
+//                             this.healthBar = scene.add.graphics();
+//                             this.maxHealth = this.attributes['health'];
+//                         }
+//                     },
+//                     updateHealthBar: jest.fn(),
+//                 }
+//                 sprite.intialize();
+
+//                 return sprite;
+//             });
+
+//             const mockScene = new (jest.requireMock('phaser').Scene)({ key: 'test' });
+//             const maxHealth = 100;
+
+//             const halfHealthFence = new SpriteItem(maxHealth, maxHealth / 2, mockScene, { layoutType: 'fence' });
+//             expect(halfHealthFence.healthBar).toBeDefined();
+//             halfHealthFence.updateHealthBar();
+//             expect(halfHealthFence.healthBar).toBeDefined();
+
+//             const fullHealthGate = new SpriteItem(maxHealth, maxHealth, mockScene, { layoutType: 'gate' });
+//             expect(fullHealthGate.healthBar).not.toBeDefined();
+//             fullHealthGate.updateHealthBar();
+//             expect(fullHealthGate.healthBar).not.toBeDefined();
+
+//         });
+//     });
```

Comment:
I would remove this section, since commented out areas just add noise to the codebase.